foodhub_picking_operations/models/stock_move.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    _order = "product_category,product_default_code"
+
+    product_category = fields.Many2one(related="product_id.categ_id", store=True)
```

Comment:
this will sort by the `id` of the category, not by its name. is it on purpose? otherwise, it should be like this:

```suggestion
    _order = "product_category_name,product_default_code"

    product_category_name = fields.Many2one(related="product_id.categ_id.name", store=True)
```

Reply:
In theory I agree with you, but in practice it works ! checkout out this [delivery](https://wholesale.test.coopiteasy.be/web#id=146&menu_id=176&cids=1&action=266&active_id=2&model=stock.picking&view_type=form), on the picking operations, valdibela is sorted last, while it's id is 3 (lower than the id from tierra y liberdad)

Reply:
Note that I will only keep the sorting for move lines, not moves

Reply:
my bad: i just checked, and indeed, odoo does it better than i think: if a many2one field is used in `_order`, it will correctly use the ordering of that model, automatically joining with all needed tables.